server/src/main/scala/ch.epfl.scala.index.server/Server.scala

```diff
@@ -60,7 +62,13 @@ object Server {
         dependencies <- api.dependencies()
         latestProjects <- api.latestProjects()
         latestReleases <- api.latestReleases()
-      } yield views.html.frontpage(keywords, targets, dependencies, latestProjects, latestReleases, userInfo)
```

Comment:
I'm anyway rebasing on top of master and rerunning scalafmt because of the latest changes upstream.